docker/.env.example

```
@@ -0,0 +1,9 @@
+# MicroMart 로컬 개발용 PostgreSQL 컨테이너 환경변수
+# docker/infra.yaml의 PostgreSQL 서비스 초기화에 사용됩니다.
+#
+# 주의: 이 값들은 로컬 개발 전용입니다.
+# 프로덕션 환경에서는 반드시 강력한 자격증명으로 변경하세요.
+
+POSTGRES_USER=micromart
+POSTGRES_PASSWORD=micromart
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)